src/doc/rustc/src/platform-support/uwp-windows-msvc.md

```diff
@@ -0,0 +1,52 @@
+# `x86_64-uwp-windows-msvc`, `i686-uwp-windows-msvc`, `thumbv7a-uwp-windows-msvc` and `aarch64-uwp-windows-msvc`
+
+**Tier: 3**
+
+Windows targets for Universal Windows Platform (UWP) applications, using MSVC toolchain.
+
+## Target maintainers
+
+- @bdbai
```

Comment:
Suggestion:

```suggestion
- [@bdbai](https://github.com/bdbai)
```

Reply:
Fixed in https://github.com/rust-lang/rust/pull/134996/commits/2389daab1b16e75216390063a1c97fccc3e0d2af